backend/Controllers/SightingsController.cs

```diff
@@ -30,4 +31,42 @@ public async Task<IActionResult> Create(SightingsRequest sightingRequest)
         }
     }
 
+    [HttpDelete("sighting={sightingId}&user={userId}/delete")]
+    public async Task<IActionResult> Delete([FromRoute] int sightingId, int userId)
+    {
+        try
+        {
+            await _service.DeleteSighting(sightingId, userId);
+            return Ok();
+        }
+        catch (UnauthorizedAccessException ex)
+        {
+            return Unauthorized(ex.Message);
+        }
+        catch (Exception ex)
+        {
+            return BadRequest(ex.Message);
+        }
+
+    }
+
+    [HttpPut("sighting={sightingId}&user={userId}/update")]
```

Comment:
for this endpoint can it actually be "{sightingId}/update&user={userId}"
So that the sightingId is part of the url and comes before hte update and the only query parameter is the userId

This way each sightingg canx have it's own update link and this will mirror the structure of the user update url too

Reply:
We'll do this! Do we need to raise a ticket to do the same thing in the delete sighting endpoint? The delete sighting currently has the url "sighting={sightingId}&user={userId}"

Reply:
yes please, good catch